pkg/filter/http/dubboproxy/dubbo.go

```diff
@@ -113,8 +112,8 @@ func (f *Filter) Decode(hc *pixiuHttp.HttpContext) filter.FilterStatus {
 	endpoint := clusterManager.PickEndpoint(clusterName, hc)
 	if endpoint == nil {
 		logger.Info("[dubbo-go-pixiu] cluster not found endpoint")
-		bt, _ := json.Marshal(pixiuHttp.ErrResponse{Message: "cluster not found endpoint"})
-		hc.SendLocalReply(http.StatusServiceUnavailable, bt)
+		errResp := pixiuHttp.ServiceUnavailable.WithError(fmt.Errorf("endpoint not found"))
```

Comment:
if the format func is not used when creating a new err, use `errors.New` instead of `fmt.Errorf`. `fmt.Errorf` only used when need format

```suggestion
		errResp := pixiuHttp.ServiceUnavailable.WithError(errors.New("endpoint not found"))
```